### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#1764]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: November 28, 2017 at 03:18AM<br><br><div style="font-family: arial,sans-serif; font-size: medium;">
<h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2iZ8BDl"><font size="3" color="#1A0DAB">Formulation of Two Stage Multiple Kernel Learning Using Regression Framework</font></a></h3>
<div><font size="2" color="#006621">SS Shiju, A Salim, S Sumitra&nbsp;- International Conference on Pattern Recognition and&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract Multiple kernel learning (MKL) is an approach to find the optimal kernel for kernel<br>
methods. We formulated MKL as a regression problem for analyzing the regression data and<br>
hence the data modeling problem involves the computation of two functions, namely, the<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizhWr" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iZ8CXV" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2Bizk4z" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iZ8E1Z"><font size="3" color="#1A0DAB">Select-and-Evaluate: A Learning Framework for Large-Scale Knowledge Graph Search</font></a>
</h3>
<div><font size="2" color="#006621">FARR Chowdhury, C Ma, MR Islam, MH Namaki&hellip;&nbsp;- Asian Conference on&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract Querying graph structured data is a fundamental operation that enables important<br>
applications including knowledge graph search, social network analysis, and cyber-network<br>
security. However, the growing size of real-world data graphs poses severe challenges for<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgB7XZ" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iZ8Ez1" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgB9iz" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2iZA1ZM"><font size="3" color="#1A0DAB">Dual Regularized Multi-view Non-negative Matrix Factorization for Clustering</font></a></h3>
<div><font size="2" color="#006621">P Luo, J Peng, Z Guan, J Fan&nbsp;- Neurocomputing, 2017</font></div>
<div><font size="2" color="#222222">Abstract Many real-world datasets are described by multiple modalities or views, which can<br>
provide compatible and complementary information to each other. Synthesizing multi-view<br>
features for data representation can lead to more comprehensive data description, which<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizmcH" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iZ8GH9" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgBd1N" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2j0MEnJ"><font size="3" color="#1A0DAB">Domain Adaptation from User-level Facebook Models to County-level Twitter Predictions</font></a>
</h3>
<div><font size="2" color="#006621">D Rieman, K Jaidka, HA Schwartz, L Ungar&nbsp;- Proceedings of the Eighth International&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract Several studies have demonstrated how language models of user attributes, such<br>
as personality, can be built by using the Facebook language of social media users in<br>
conjunction with their responses to psychology questionnaires. It is challenging to apply<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgBf9V" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYW6aZ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizpFp" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iYmHVu"><font size="3" color="#1A0DAB">Learning Deep Semantic Embeddings for Cross-Modal Retrieval</font></a>
</h3>
<div><font size="2" color="#006621">C Kang, S Liao, Z Li, Z Cao, G Xiong&nbsp;- Asian Conference on Machine Learning, 2017</font></div>
<div><font size="2" color="#222222">Abstract Deep learning methods have been actively researched for cross-modal retrieval,<br>
with the softmax cross-entropy loss commonly applied for supervised learning. However, the<br>
softmax cross-entropy loss is known to result in large intra-class variances, which is not not<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizqJt" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYmIZy" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgBhP5" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2iYmK3C"><font size="3" color="#1A0DAB">Neural versus Phrase-Based MT Quality: an In-Depth Analysis on English-German and English-French</font></a></h3>
<div><font size="2" color="#006621">L Bentivogli, A Bisazza, M Cettolo, M Federico&nbsp;- Computer Speech &amp; Language, 2017</font></div>
<div><font size="2" color="#222222">Abstract Within the field of statistical machine translation, the neural approach (NMT) is<br>
currently pushing ahead the state of the art performance traditionally achieved by phrase-<br>
based approaches (PBMT), and is rapidly becoming the dominant technology in machine<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2Bizskz" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYoQAN" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizsRB" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iZA2gi"><font size="3" color="#1A0DAB">Supervised Attention for Sequence-to-Sequence Constituency Parsing</font></a>
</h3>
<div><font size="2" color="#006621">H Kamigaito, K Hayashi, T Hirao, H Takamura&hellip;&nbsp;- Proceedings of the Eighth&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract The sequence-to-sequence (Seq2Seq) model has been successfully applied to<br>
machine translation (MT). Recently, MT performances were improved by incorporating<br>
supervised attention into the model. In this paper, we introduce supervised attention to<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BgBj9F" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYmMbK" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2Bizx7R" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iYmNMQ"><font size="3" color="#1A0DAB">Domain Adaptation for Relation Extraction with Domain Adversarial Neural Network</font></a>
</h3>
<div><font size="2" color="#006621">L Fu, TH Nguyen, B Min, R Grishman&nbsp;- Proceedings of the Eighth International Joint&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract Relations are expressed in many domains such as newswire, weblogs and phone<br>
conversations. Trained on a source domain, a relation extractor's performance degrades<br>
when applied to target domains other than the source. A common yet labor-intensive method<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizxVp" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iX5DPQ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BizA3x" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iX5FqW"><font size="3" color="#1A0DAB">Term extraction and document similarity in an Integrated Learning Design Environment</font></a>
</h3>
<div><font size="2" color="#006621">A Mart&iacute;nez Rodr&iacute;guez - 2017</font></div>
<div><font size="2" color="#222222">The Integrated Learning Design Environment is a social platform focused in supporting<br>
teachers in the computer-assisted design of Learning activities. In this platform, teachers and<br>
course designers can contextualize, author and share their designs within their community.<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BhzuJG" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYmQZ2" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BjaSAc" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2iX5ITE"><font size="3" color="#1A0DAB">Feature Map Pooling for Cross-View Gait Recognition Based on Silhouette Sequence Images</font></a>
</h3>
<div><font size="2" color="#006621">Q Chen, Y Wang, Z Liu, Q Liu, D Huang</font></div>
<div><font size="2" color="#222222">Abstract In this paper, we develop a novel convolutional neural network based approach to<br>
extract and aggregate useful information from gait silhouette sequence images instead of<br>
simply representing the gait process by averaging silhouette images. The network takes a<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BiKxSX" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2iYmUrK" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BjA6hX" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><hr noshade size="1">
<font size="2" color="#222222">This Google Scholar Alert is brought to you by Google.<br><br><a href="http://ift.tt/2vUKjPi"><font color="#1A0DAB">Cancel alert</font></a><br><a href="http://ift.tt/2eWhNpX"><font color="#1A0DAB">List my alerts</font></a></font>
</div>
<br><br>
via Gmail
